### PR TITLE
CASSANDRA-16788 Published blog Changelog #8 June 2021

### PR DESCRIPTION
**Pull request by ErickRamirezAU**
Highlights:
- Apache Cassandra 4.0-rc2 released
- say hello to our Google Summer of Code intern
- new community intro to Cassandra videos

### CODE REVIEW

**Comment by michaelsembwever** [comment]
Staged with 5f5df6777
https://cassandra.staged.apache.org/blog/Apache-Cassandra-Changelog-8-June-2021.html

File: content/blog/Apache-Cassandra-Changelog-8-June-2021.html [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+
+					<h4 id="added">Added</h4>
+
+					<p>We’ve released new Apache Cassandra community videos: <a target="_blank" href="https://www.youtube.com/watch?v=CKdT6f6wQyo">What is Apache Cassandra?</a> and <a target="_blank" href="https://www.youtube.com/watch?v=kOf29M8qFts">What's New in Apache Cassandra 4.0?</a>. A big thank you goes to Ben Bromhead, Ekaterina Dimitrova, and Patrick McFadin for presenting and helping put these together! &#128079; Remember, if you weren’t able to attend Cassandra World Party, we’ve also uploaded all the recorded lightning talks to this <a target="_blank" href="https://www.youtube.com/watch?v=TZNrTlIazA0&list=PLqcm6qE9lgKLi4TD3azt3soaCjfXte32m">playlist</a>.</p>
+
+					<iframe width="560" height="315" src="https://www.youtube.com/watch?v=kOf29M8qFts" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (ErickRamirezAU) [inline content/blog/Apache-Cassandra-Changelog-8-June-2021.html:228]: The `<iframe>` does not render correctly so will need to be redone.

**Review by ErickRamirezAU — COMMENTED** [review]
The `<iframe>` doesn't render on the site so I'll need to redo it.

I've also just realised that the entry is missing from the blog listings in `/blog`.

**Comment by ErickRamirezAU** [comment]
Made necessary changes in PR #63. Closing this PR.

**Pull request closed without merging**